get_servers_info.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/python
 
+import os
 import argparse
-import os.path
```

Comment:
I'm just curious why you moved this import up. I usually just sort imports alphabetically.

Reply:
you are right.